lib/Runtime/Base/FunctionBody.h

```diff
@@ -591,6 +591,9 @@ namespace Js
 #endif
     public:
         uint frameHeight;
+#if ENABLE_NATIVE_CODEGEN
+        bool nativeEntryPointProcessed;
```

Comment:
Do you know how we got away with having this only on FunctionEntryPointInfo? Does it never get used on loop entry points?